src/controllers/tenants.cr

```diff
@@ -63,7 +65,7 @@ class Tenants < Application
   # returns the limits for the current domain (Host header)
   @[AC::Route::GET("/current_limits")]
   def current_limits : Limits
-    response.headers["X-Delegated"] = (!!current_tenant.delegated).to_s
+    response.headers["X-Delegated"] = (!!current_tenant.delegated).to_s.gsub(/[\r\n]+/, String.new)
```

Comment:
don't think there could possibly be an issue here
We're calling: `Bool#to_s`

Reply:
Completely agree, thought would be a precaution to do so and now that I think about it you can not really manipulate the bool out of true or false.